MapboxNavigation/NavigationMapView.swift

```diff
@@ -1104,19 +1108,39 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      Sets the camera directly over a series of coordinates.
      */
     @objc public func setOverheadCameraView(from userLocation: CLLocationCoordinate2D, along coordinates: [CLLocationCoordinate2D], for bounds: UIEdgeInsets) {
-        let slicedLine = Polyline(coordinates).sliced(from: userLocation).coordinates
-        let line = MGLPolyline(coordinates: slicedLine, count: UInt(slicedLine.count))
-        
-        tracksUserCourse = false
-        let camera = self.camera
-        camera.pitch = 0
-        camera.heading = 0
-        self.camera = camera
+//        let slicedLine = Polyline(coordinates).sliced(from: userLocation).coordinates
+//        let line = MGLPolyline(coordinates: slicedLine, count: UInt(slicedLine.count))
+//
+//        tracksUserCourse = false
+//        let camera = self.camera
+//        camera.pitch = 0
+//        camera.heading = 0
+//        self.camera = camera
+//
+//        // Don't keep zooming in
+//        guard line.overlayBounds.ne.distance(to: line.overlayBounds.sw) > NavigationMapViewMinimumDistanceForOverheadZooming else { return }
+//
+//        setVisibleCoordinateBounds(line.overlayBounds, edgePadding: bounds, animated: true)
+//        self.animView?.percent = 0
+//        UIView.animate(withDuration: SWViewController.kAnimDuration) {
+//            self.animView?.percent = 100.0
+//        }
+        
+        self.navigationCamera.altitude = 100
+        self.navigationCamera.pitch = 50
+        self.navigationCamera.course = self.direction
+        
+        UIView.animate(withDuration: 1) {
+            self.navigationCamera.altitude = 10_000
+        }
         
-        // Don't keep zooming in
-        guard line.overlayBounds.ne.distance(to: line.overlayBounds.sw) > NavigationMapViewMinimumDistanceForOverheadZooming else { return }
+        UIView.animate(withDuration: 1, delay: 1, options: [], animations: {
+            self.navigationCamera.pitch = 0
+        }, completion: nil)
         
-        setVisibleCoordinateBounds(line.overlayBounds, edgePadding: bounds, animated: true)
+        UIView.animate(withDuration: 1, delay: 2, options: [], animations: {
+            self.navigationCamera.course = 0
+        }, completion: nil)
```

Comment:
How are you figuring out the proper center coordinate here?

Reply:
Center coordinate was added in https://github.com/mapbox/mapbox-navigation-ios/pull/1240/commits/8d33ef76bb9340d7c7ec2d70c2c04cfee2d3b539. Next up is visible bounds.